resources/analysisTools/indelCallingWorkflow/checkSampleSwap_TiN.pl

```diff
@@ -270,7 +271,8 @@
   my $end_col = $columnCounter+6;
   my $gnomAD_genome_col = $columnCounter+8;
   my $gnomAD_exome_col = $columnCounter+9;
-  my $localcontrol_col = $columnCounter+10;
+  my $localcontrol_wgs_col = $columnCounter+10;
+  my $localcontrol_wes_col = $columnCounter+11;
```

Comment:
You may remember that we had a discussion about how to deal with added columns in output files with respect to version numbers of the plugin? To me it seemed the discussion ended without a clear decision. Here we have a new output column appended to a table as new last column.

Would you rather increase the major version, or would be a minor version bump fine?

Reply:
Minor version bump should be good in this case, have added a changelog.